torch_xla/csrc/tensor_methods.cpp

```diff
@@ -392,6 +392,13 @@ void all_reduce(const std::vector<XLATensorPtr>& inputs,
   }
 }
 
+XLATensorPtr all_reduce(const XLATensorPtr& input, AllReduceType reduce_type,
```

Comment:
can you call it `all_reduce _no_token`, the only difference in signature is it does not take `pin_layout` but the main difference in the op is that it does not set token.. It is better to reflect that in the name.

Reply:
Sure. I can follow up with that.